demo-terraform/aws/wordpress_ec2_alb_rds/alb.tf

```diff
@@ -1,53 +1,58 @@
 locals {
-    lb_name = "alb${random_string.lab_id.result}"
-    tg_name = "tg${random_string.lab_id.result}"
+  lb_name = "alb${random_string.lab_id.result}"
+  tg_name = "tg${random_string.lab_id.result}"
 }
 
 resource "aws_lb" "web" {
```

Comment:
**Accurics remediation**
 
**[LOW]: Ensure HTTP2 is enabled for AWS LB (Load Balancer)**
 
**Vulnerability**
AWS application load balancer has HTTP2 disabled. This may lead to slow rendering and page loading with high latency.
 
**Remediation**
Classic Load Balancing uses Secure Sockets Layer (SSL) negotiation configurations, known as security policies, to negotiate connections between the clients and the load balancer. When you use HTTPS/SSL for your front-end connections, you can use either a predefined security policy or a custom security policy. You must deploy an SSL certificate on your load balancer.

source: accurics-